src/lib/pages/frontsite/PatronProfile/PatronShowLink.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Icon, Divider } from 'semantic-ui-react';
+
+export const PatronShowLink = ({
+  items,
+  onShowAllClick,
+  onShowLessClick,
+  rowsPerPage,
+}) => {
+  const allHits = items.total === items.hits.length ? true : false;
+  return (
+    items.total > 0 &&
+    items.total > rowsPerPage && (
+      <Grid>
+        <Grid.Column textAlign="center">
+          {!allHits ? (
+            <>
+              <Divider hidden />
+              <div className="link-like" onClick={() => onShowAllClick()}>
```

Comment:
No link in the end? Did you see my comment about `accessbility`?

Reply:
I understood from your comment that we don't want the <a> tag there because of the screen readers, but now I think I misunderstood